plugin/src/test/java/org/opensearch/ml/sdkclient/TestDataObject.java

```diff
@@ -0,0 +1,48 @@
+package org.opensearch.ml.sdkclient;
```

Comment:
Add license

Reply:
Would be nice if spotless added the licenses. :|